### PR TITLE
yoga: Build kayobe image on push, make TF aio volume configurable

### PR DESCRIPTION
**Pull request by markgoddard**
- CI: Build stackhpc-kayobe-config image on push to release branches
- Terraform aio: Make instance volume size configurable

### CODE REVIEW

File: .github/workflows/stackhpc-build-kayobe-image.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@
 name: Build kayobe image
 
 on:
+  push:
+    branches:
+      # NOTE(upgrade): Reference only the current release branch here.
+      - stackhpc/zed
```
Comment (Alex-Welsh) [inline .github/workflows/stackhpc-build-kayobe-image.yml]: Not the current release

**Review by Alex-Welsh — CHANGES_REQUESTED** [review]

**Review by Alex-Welsh — DISMISSED** [review]

File: .github/workflows/stackhpc-build-kayobe-image.yml (hunk 1 above)
Comment (markgoddard) [reply .github/workflows/stackhpc-build-kayobe-image.yml]: :facepalm:

**Comment by markgoddard** [comment]
I'll stop fiddling now...

**Review by Alex-Welsh — APPROVED** [review]

**Comment by markgoddard** [comment]
aios passing (when they can get a VM)

**Pull request merged**